src/commands/prompt.ts

```diff
@@ -1,14 +1,43 @@
-import {logger} from "../logger";
-import {SlashCommandBuilder} from "@discordjs/builders";
-import {CommandInteraction, CacheType} from "discord.js";
-import {CommandType} from "../types";
+import { logger } from "../logger";
+import { inlineCode, SlashCommandBuilder, SlashCommandStringOption } from "@discordjs/builders";
+import { CommandInteraction, CacheType, TextChannel, MessageEmbed } from "discord.js";
+import { CommandType } from "../types";
+import { Config } from "../config";
 
 const promptModule: CommandType = {
-  data: new SlashCommandBuilder().setName("prompt").setDescription("Prompt?"),
+  data: new SlashCommandBuilder()
+    .setName("prompt")
+    .setDescription("Ask something?")
+    .addStringOption((opt: SlashCommandStringOption) =>
+      opt
+        .setName("question")
+        .setDescription("What is the question?")
+        .setRequired(true)
+    ),
   execute: async (interaction: CommandInteraction<CacheType>) => {
-    logger.info("hello from prompt");
-    await interaction.reply("hello from prompt");
+
+    let configChannel = Config?.prompt.channel
+    let channelId = interaction.guild?.channels.cache.find(channel => channel.name === configChannel) as TextChannel
+
+    if (channelId === undefined) {
+      await interaction.reply({ content: `Channel ${inlineCode(configChannel!.toString())} does not exist in this guild` })
+      return;
+    }
+    let question = interaction.options.getString("question")!;
+
+    let promptMess = Config?.prompt.top_text + question + Config?.prompt.bottom_text
```

Comment:
I believe `top_text` and `bottom_text` should have two newlines between them and the question. @prairir please correct me if I'm wrong.

Example:
![image](https://user-images.githubusercontent.com/52428953/186442967-1fa5ea91-f32c-41ab-bd05-26b587bf2953.png)

The actual question is `What project of yours are you most proud of?`, and the green boxes are top/bottom text. Notice the newlines between each.

Reply:
The newlines are user added. We wanted it to be flexible.